src/app/screens/Home/DefaultView/index.tsx

```diff
@@ -91,7 +91,7 @@ const DefaultView: FC<Props> = (props) => {
         if (e instanceof Error) toast.error(`Error: ${e.message}`);
       }
     })();
-  }, []);
+  }, [account]);
```

Comment:
should that account variable in line 82 be called differently to not overlay the variable?

Reply:
it won't effect in any way as useEffect doesn't take variables as dependency defined within useEffect itself. 

but still i changed the name of the local variable